packages/service-library/tests/long_running_tasks/test_long_running_tasks_task.py

```diff
@@ -159,9 +159,9 @@ async def unique_task(task_progress: TaskProgress):
     start_task(tasks_manager=tasks_manager, task=unique_task, unique=True)
 
     # ensure unique running task regardless of how many times it gets started
-    for _ in range(5):
-        with pytest.raises(TaskAlreadyRunningError) as exec_info:
-            start_task(tasks_manager=tasks_manager, task=unique_task, unique=True)
+    with pytest.raises(TaskAlreadyRunningError) as exec_info:
+        start_task(tasks_manager=tasks_manager, task=unique_task, unique=True)
+    assert "must be unique, found: " in f"{exec_info.value}"
```

Comment:
not sure you are aware and this is minor, but pytest.raises can take an optional parameter to match the error message